doc/api/pyplot_summary.rst

```diff
@@ -142,6 +241,10 @@ Axis configuration
    ylim
    yscale
    yticks
+   figlegend
+   legend
```

Comment:
The legend entries should probably be in Basic/Core?

Reply:
Nope. Basic/Core is a subsection of "Adding data to the plot".

Moved  to "Text and annotations"

Reply:
Not fully sold, but sold enough to merge.